exec-unittests

```diff
@@ -161,43 +166,57 @@ if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
     fi
 fi
 
-_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
+_agent_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
 
 _para_jobs_file="${_toxenvdir}/agent-legacy-jobs.lis"
 trap "rm -f ${_para_jobs_file}" EXIT INT TERM
 
 for _major in ${major_list}; do
-    # Verify the Agent or Server Makefile functions correctly.
-    if [[ "${_major}" == "agent" || "${_major}" == "server" ]]; then
-        verify_make_source_tree ${_major} || rc=1
-    fi
-
     if [[ "${_major}" == "agent" ]]; then
-        # The parallel program is really cool.  The usage of `parallel` is
-        # internal and automated; only test code depends on this tool, and we,
-        # as developers, have viewed the citation and are justified in
-        # suppressing future displays of it in our development processes (use of
-        # --will-cite below).
-
-        let count=0
-        for _subtst in ${_subtst_list}; do
+        if [[ "${subtst:-legacy}" == "python" ]]; then
+            continue
+        fi
+        for _subtst in ${_agent_subtst_list}; do
             if [[ "${subtst:-legacy}" == "legacy" || "${subtst}" == "$(basename ${_subtst})" ]]; then
                 echo "${_major} ${_subtst} ${posargs}"
-                (( count++ ))
             fi
         done > ${_para_jobs_file}
-        parallel --will-cite -k --lb -a ${_para_jobs_file} ${para_jobs_arg} para_run_legacy
-        if [[ ${?} -ne 0 ]]; then
-            rc=1
-        fi
-        if [[ ${count} -eq 0 && "${subtst}" != "python" ]]; then
+        if [[ -s ${_para_jobs_file} ]]; then
+            # The parallel program is really cool.  The usage of `parallel` is
+            # internal and automated; only test code depends on this tool, and we,
+            # as developers, have viewed the citation and are justified in
+            # suppressing future displays of it in our development processes (use of
+            # --will-cite below).
+            if [[ ${_ECHO} == "echo" ]]; then
+                cat ${_para_jobs_file}
+            fi
```

Comment:
I recommend `-n ${_ECHO}` instead of comparing its value to `echo`.

And, I would also place this `if` statement before the block comment.  (We test `_para_jobs_file` on the previous line, and the comment refers to the line after this block, so let's move this block out of the way.)